client/lib/domain/auth/value_validators.dart

```diff
@@ -34,8 +34,9 @@ Either<ValueFailure<String>, String> validateName(String nameStr) {
 }
 
 Either<ValueFailure<String>, String> validateRole({required String roleStr}) {
-  if (!(roleStr == 'User' || roleStr == 'Admin')) {
```

Comment:
@MichaelAlemayehu1 is using the capitals when registering. Might wanna change that too

Reply:
will do